src/PiWeb.Import.Sdk/ImportData/AdditionalFileSourceDataItem.cs

```diff
@@ -27,13 +27,13 @@ namespace Zeiss.PiWeb.Import.Sdk.ImportData;
 
 /// <summary>
 /// <inheritdoc />
-/// This implementation returns a stream from a given <see cref="IFileSource"/>.
+/// This implementation returns a stream from a given <see cref="IImportFile"/>.
 /// </summary>
 public sealed class AdditionalFileSourceDataItem : AdditionalDataItem

```

Comment:
Do we want to keep "FileSource" in the class name?

Reply:
Ah nice catch, I missed that one.